src/main/java/com/github/hanyaeger/api/engine/entities/entity/YaegerEntity.java

```diff
@@ -352,4 +352,30 @@ public void addToParent(final EntityProcessor processor) {
     public RotationBuffer getRotationBuffer() {
         return rotationBuffer;
     }
+
+    /**
+     * Calculate the absolute location of this {@link YaegerEntity} in the scene.
+     * Because {@link CompositeEntity} uses a relative coordinate system, this method is needed to make calculations
+     * for entities that are part of a {@link CompositeEntity}.
+     * @return a {@link Coordinate2D} with the absolute coordinates of the {@link YaegerEntity}.
+     */
+    protected Coordinate2D getLocationInScene() {
```

Comment:
Is this the best place for this method? Right now most behaviour is placed as default methods in various interfaces. It feels as if this method should be placed in one of those interfaces.